anatomy/tracks/config-json.md

```diff
@@ -19,6 +19,24 @@ The following top-level properties contain general track metadata:
   - `test_runner`: a `boolean` value indicating if a [test runner](../track-tooling/test-runners/README.md) has been implemented (required)
   - `representer`: a `boolean` value indicating if a [representer](../track-tooling/representers/README.md) has been implemented (required)
   - `analyzer`: a `boolean` value indicating if an [analyzer](../track-tooling/analyzers/README.md) has been implemented (required)
+- `files`: The patterns for the locations of the files used in this exercise. (optional)
+  - `solution`: stub implementation file(s) pattern (optional)
+  - `test`: test file(s) pattern (optional)
+  - `example`: example implementation file(s) pattern (optional
+  - `exemplar`: exemplar implementation file(s) pattern (optional)
+
+### Files
+
+This key is used to specify track-wide file locations. Rather than maintainers having to manually set the `files` key in the _exercise_'s `config.json` files, support will be added to [configlet](./README.md) to use these track-wide patterns to automatically populate the exercise's `files` key.
```

Comment:
```suggestion
This key is used to specify track-wide file locations. Rather than maintainers having to manually set the `files` key in the _exercise_'s `config.json` files, support will be added to [configlet](./README.md) to use these track-wide patterns to automatically populate the exercises' `files` key.
```